### DISCUSSION TITLE
Why is Forum Search not working?

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#1202] [Help] [answered]
Due to a 0-day security exploit found on dependencies Forum Search is currently disabled. This is to protect your account. We will fix this issue soon.

**Comment by mxmou** [comment]
Does this apply to all addons using ScratchDB?

**Reply by apple502j** [reply]
Only forum search, for now.

**Reply by jeffalo** [reply]
yes all addons using scratchdb do not function as intended

**Comment by cobaltt7** [comment]
Looking at [this](https://github.com/ScratchAddons/ScratchAddons/pull/1204#issuecomment-754223311), can the issue for this be closed?

**Reply by WorldLanguages** [reply]
No, that hasn't reached stable yet

**Comment by Marc92020** [comment]
The forum search works for me so...

**Reply by Marc92020** [reply]
I was wrong

**Comment by cobaltt7** [comment]
Is that why post counts dosen’t work anymore?

**Reply by mxmou** [reply]
Yes. As Jeffalo said, addons using ScratchDB don't currently work.

**Comment by apple502j** [comment, marked as answer]
This issue is now fixed.
